### PR TITLE
Fix : some comment and translation

### PR DESCRIPTION
**Pull request by grandoc**

### CODE REVIEW

File: htdocs/accountancy/admin/account.php [hunk 1]
```diff
@@ -272,8 +272,17 @@
 	if ($limit > 0 && $limit != $conf->liste_limit) $param .= '&limit='.$limit;
 	if ($search_account) $param .= '&search_account='.urlencode($search_account);
 	if ($search_label) $param .= '&search_label='.urlencode($search_label);
+<<<<<<< HEAD
```
Comment (aspangaro) [inline htdocs/accountancy/admin/account.php:275]: Hello @grandoc , you seems to have a merge problem, can you fix it ?

File: htdocs/includes/tecnickcom/tcpdf/fonts/raleway.php [hunk 2]
```diff
@@ -0,0 +1,74 @@
+<?php
```
Comment (eldy) [inline htdocs/includes/tecnickcom/tcpdf/fonts/raleway.php:1]: It seems your PR add some font files that are not part of tcpdf into tcpdf dir.

**Comment by eldy** [comment]
There is still files for "roboto"

**Comment by grandoc** [comment]
Ok Laurent
I have fixed it

**Pull request merged**